camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java

```diff
@@ -363,8 +356,10 @@ public static boolean isReferenceParameter(String parameter) {
                 // The bean is a list
                 return (List) bean;
             } else {
-                // The bean is a list element
-                return Arrays.asList(elementType.cast(bean));
+            	// The bean is a list element
```

Comment:
There seems to be a tab character here, so the Checkstyle check fails with:

    camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java:359:13: File contains tab characters (this is the first instance). [FileTabCharacter]

Can you fix that? You can check with:

    mvn -Psourcecheck verify